grasa_event_locator/settings/base.py

```diff
@@ -27,7 +27,7 @@
 SITE_ID = 1
 
 # https://docs.djangoproject.com/en/2.2/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["grasa.larrimore.de", "abba.larrimore.de"]
```

Comment:
These hosts should be set via environment variables passed to the container on start time. I think it is better to leave this blank in order to ship an empty default configuration.

Reply:
you're right, which is why I just created #12 but in the meantime, this should stay until we can resolve that issue.

Reply:
@nlMeminger Okay, fair. Instead of putting this in the `base` configuration, could you please add it to the development environment settings in `grasa_event_locator/settings/development.py`? This way it won't be inherited in a production deployment.

Reply:
I think for now it should stay until the issue is resolved since this is needed in order to access the site from a url.  This can always be changed at a later time as long as it's noted.